config.default

```diff
@@ -8,7 +8,7 @@ COMMITTER_EMAIL="ada@example.com"
 # COMMITTER_UID=1000
 # COMMITTER_GID=1000
 
-## Planned release date (optional; default="thursday")
+## Planned release date (optional; default="thursday"; format=YYYY-MM-DD)
```

Comment:
More precisely, the date format is as defined by the GNU date command-line utility, see also: `man date`

Reply:
Let's leave as it is then.